src/fetchers/book.ts

```diff
@@ -0,0 +1,28 @@
+import { useQuery } from '@tanstack/react-query';
+import { Position } from '@penumbra-zone/protobuf/penumbra/core/component/dex/v1/dex_pb';
+
+interface BookResponse {
+  asks: Position[];
+  bids: Position[];
+}
+
+export const useBook = (
+  symbol1: string | undefined,
+  symbol2: string | undefined,
+  hops: number | undefined,
+  limit: number | undefined,
+) => {
```

Comment:
idea: Not critical to this PR, but starting to realize we'll want to include block height in the query key with nearly all our queries (as they'll get invalidated every block and refetched). Maybe something for later.